tools/run-tests.py

```diff
@@ -108,7 +108,7 @@ def skip_if(condition, desc):
 
 # Test options for test262-esnext
 TEST262_ESNEXT_TEST_SUITE_OPTIONS = [
-    Options('test262_tests_esnext', OPTIONS_PROFILE_ESNEXT + ['--line-info=on', '--error-messages=on']),
+    Options('test262_tests_esnext', OPTIONS_PROFILE_ESNEXT + ['--line-info=on', '--error-messages=on', '--compile-flag=-m32', '--system-allocator=on']),
```

Comment:
You can use 32 bit cpointers on on 64 bit as well, so you don't need to disable tests.

Reply:
Good idea, I'll check it with cpointer and larger heap limit.